ci-operator/populate-secrets-from-bitwarden.sh

```diff
@@ -212,6 +212,14 @@ update_secret generic "cluster-secrets-${target_cloud}"                       \
 	"$( format_field_value telemeter "Telemeter Token" "telemeter-token" )"
 oc label secret "cluster-secrets-${target_cloud}" "ci.openshift.io/managed=true"
 
+target_cloud="openstack"
+update_secret generic "cluster-secrets-${target_cloud}"                       \
+	--from-literal=pull-secret="$(merge_pull_secrets)"                          \
+	"$( format_attachment "openstack" clouds.yaml )"                            \
+	"$( format_attachment "openstack" clouds.yaml credentials )"                \
+	"$( format_attachment "openstack" ssh-publickey )"
```

Comment:
I think we're missing the private key

Reply:
I also found that strange, but it is also missing from the secret currently in the cluster:

```yaml
apiVersion: v2
data:
  clouds.yaml: XXX
  credentials: XXX (same as clouds.yaml)
  pull-secret: XXX
  ssh-publickey: XXX
kind: Secret
metadata:
  name: cluster-secrets-openstack
  namespace: ci
type: Opaque
```

Reply:
Can we use the same values as for AWS? That way we can make ssh'ing into this CI environments more consistent. I believe you guys already have a shared private key that you use for this envs